tests/sentry/backup/test_models.py

```diff
@@ -61,3 +62,19 @@ def test_organization(self):
         user = self.create_user()
         self.create_organization(owner=user)
         self.import_export_then_validate()
+
+    def test_actor(self):
```

Comment:
Can you merge https://github.com/getsentry/sentry/pull/52690 and then decorate all of the new `test_...` methods with the `@targets_models(...)` decorator that PR introduces? It just checks that whatever model we claim to be testing actually ends up in the final output, and also enables future automated completeness checks down the line.